apps/condo/domains/notification/constants/constants.js

```diff
@@ -819,6 +927,13 @@ const MESSAGE_DELIVERY_OPTIONS = {
         isAllowedToChangeDefaultTransport: false,
         isVoIP: true,
     },
+    [CANCELED_CALL_MESSAGE_PUSH_TYPE]: {
+        priority: MESSAGE_DELIVERY_FAST_PRIORITY,
+        allowedTransports: [PUSH_TRANSPORT],
+        defaultTransports: [PUSH_TRANSPORT],
+        isAllowedToChangeDefaultTransport: false,
+        pushType: PUSH_TYPE_SILENT_DATA,
```

Comment:
I'd have preffered it called preferredPushType to indicate that this one will overwrite the one in RemoteClient, but its not a deal breaker for me